src/server/api/init/init.ts

```diff
@@ -1,21 +1,10 @@
-import { Response } from 'express'
+import { Request, Response } from 'express'
 
-import { InitRequest } from '@meta/api/request'
-
-import { AssessmentController } from '@server/controller/assessment'
-import { SettingsController } from '@server/controller/settings'
 import Requests from '@server/utils/requests'
 
-export const init = async (req: InitRequest, res: Response) => {
+export const init = async (req: Request, res: Response) => {
```

Comment:
Do we need Init anymore?
Should we separate the concerns to specific endpoints?
Eg here we just need user
GET /user or similar